src/ape/api/networks.py

```diff
@@ -333,7 +333,8 @@ def default_network_name(self) -> str:
 
         elif len(self.networks) >= 1:
             # Use the first network.
-            return self.networks[0]
+            key = next(iter(self.networks.keys()))
```

Comment:
What python version made dict key order not random?

Reply:
3.6